.travis/README.md

```diff
@@ -0,0 +1,185 @@
+
+[Travis CI](https://travis-ci.org/) is a continuous integration platform that automatically builds, tests, and deploys your application whenever changes are committed to the codebase. It is also used to manage automatic deployment and notifications. Open-source projects on Travis are free.
+
+[Fly](https://fly.io/) is a Docker-based platform as a service (PaaS) that allows you to deploy globally-distributed applications on the edge. Being location-aware, Fly enables you to serve your application from the edge node closest to your users. Most applications that can be packaged as a Docker image can be deployed to Fly.
+
+## How to Configure Travis CI to Deploy Your Application to Fly
+
+In this tutorial, you’ll learn how to deploy an application to Fly.io using Travis CI. You’ll see how to connect your application’s repository, and walk through the steps to configure Travis CI to build and deploy your application to Fly.
+
+### Prerequisites
+
+- A [GitHub](https://github.com/) or [Bitbucket](https://bitbucket.org/) account.
+- The [Flyctl command line tool](https://fly.io/docs/flyctl/installing/).
+- (Optional) [Docker](https://www.docker.com/) for testing the demo application locally.
+
+### Cloning the Sample Application
+
+To demonstrate deploying an application to Travis, you can use the [Flygreeting sample app on GitHub](https://github.com/fly-examples/flygreeting).
+
+Fork [flygreeting](https://github.com/fly-examples/flygreeting) to your own repository (if you use GitHub), then clone it to your local machine:
+
+```
+git clone git@github.com:fly-examples/flygreeting.git
+cd flygreeting
+```
+
+If you use Bitbucket, you can `git push` the cloned repository to your Bitbucket account.
+
+To test the Flygreeting app locally, you can run it using Docker:
+
+```
+docker build -t flygreeting .
+docker run -p 8000:8080 -d flygreeting
+```
+
+Visit your application in the browser at `http://localhost:8000/v1/countries`. You should see a JSON response from the Flygreeting application.
+
+### Creating the Fly Configuration file
+You'll need to use `flyctl` to create a new application and configuration file. If you haven't already, install the appropriate version of `flyctl` for your operating system using the [instructions here](https://fly.io/docs/hands-on/installing/).
+
+Next, [sign up](https://fly.io/docs/hands-on/sign-up/) or [sign in](https://fly.io/docs/hands-on/sign-in/) to your Fly account via the command line:
+
+```bash
+# Sign up
+flyctl auth signup
+
+# Or sign in
+flyctl auth login
+```
+
+You will be directed to a web page that will allow you to log in using your GitHub account or email and password.
+
+Every Fly application makes use of the `fly.toml` file to manage deployments. Because you cloned this demo app, it already has a `fly.toml` file. You can delete it and generate a new one automatically by running:
+
+```
+rm fly.toml && flyctl apps create
```

Comment:
Preferred command for creating an app is `flyctl init` - apps create is retained for backwards compatibility.